### PR TITLE
Dockerized the Web App

### PR DESCRIPTION
**Pull request by DanBrown47**
I have created the `Dockerfile` and `.dockerignore` files. Which are tested and found perfectly working

In order to test functioning follow the updates in `Readme` file

`.env.save` file is created for `.env` reference 
Working of the docker file
Docker build successful statement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +105,17 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Migrate the changes to the database `$ python manage.py migrate`
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
- 
+
+ ### Setting-up the project in docker
+    Docker  use OS-level virtualization to deliver software in packages called containers. Containers are isolated from one another and bundle their own software, libraries and configuration files. For instance when you want to contribute to this project and you have a fresh OS or you are working from a alien system or environment. You can directly proceed to contribute to code. Docker will take care of the rest like setting up a fully functional environment for testing. 
+  * Clone the repository to your local machine `$ git clone https://github.com/<your-github-username>/JagratiWebApp.git`
+  * Build the docker file to an image `sudo docker build -t IMAGE_NAME .`
```
Comment (garg3133) [inline README.md:112]: Shouldn't you change the directory to `JagratiWebApp` before running this comment?

File: README.md [hunk 2]
```diff
@@ -105,7 +105,17 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Migrate the changes to the database `$ python manage.py migrate`
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
- 
+
+ ### Setting-up the project in docker
+    Docker  use OS-level virtualization to deliver software in packages called containers. Containers are isolated from one another and bundle their own software, libraries and configuration files. For instance when you want to contribute to this project and you have a fresh OS or you are working from a alien system or environment. You can directly proceed to contribute to code. Docker will take care of the rest like setting up a fully functional environment for testing.
```
Comment (garg3133) [inline README.md:110]: This paragraph do not appear properly in Readme.

File: README.md [hunk 3]
```diff
@@ -105,7 +105,17 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
...
- 
+
+ ### Setting-up the project in docker
+    Docker  use OS-level virtualization to deliver software in packages called containers. Containers are isolated from one another and bundle their own software, libraries and configuration files. For instance when you want to contribute to this project and you have a fresh OS or you are working from a alien system or environment. You can directly proceed to contribute to code. Docker will take care of the rest like setting up a fully functional environment for testing. 
+  * Clone the repository to your local machine `$ git clone https://github.com/<your-github-username>/JagratiWebApp.git`
+  * Build the docker file to an image `sudo docker build -t IMAGE_NAME .`
+  * Run the docker image `sudo docker run -p 8000:8000 image_name`
+  * The server will start at default port, head to web browser to test
+
+ **Note:** Docker container will copy the files in the repository, One should always do proceduers in setting up project like configuring `.env` file and `sample-db.sqlite3` renaming.
```
Comment (garg3133) [inline README.md:116]: Can you make these points more clear? As people who'll come to this section won't probably read the above section.

File: README.md (hunk 2 above)
Comment (DanBrown47) [reply README.md]: I have removed the paragraph

File: README.md (hunk 3 above)
Comment (DanBrown47) [reply README.md]: I have given a more concise explanation, feel free to make changes or update me 👍🏽

**Comment by garg3133** [comment]
One more thing, shouldn't we use python version 3.7 or 3.8 for this? Python 3.6 seems too old 😅

**Comment by DanBrown47** [comment]
@garg3133 I have resolved the versioning

**Comment by garg3133** [comment]
Merged after making a few changes to README.

Thanks @DanBrown47 for your contribution 🚀

**Pull request merged**